OpenKeychain/src/main/java/org/sufficientlysecure/keychain/service/KeychainService.java

```diff
@@ -653,7 +665,7 @@ public void setPreventCancel() {
     }
 
     public void serialKeyImport(ArrayList<ParcelableKeyRing> keyList, final String keyServer,
-                                ProviderHelper providerHelper) {
+                                ProviderHelper providerHelper, Proxy proxy) {
         Log.d(Constants.TAG, "serial key import starting");
         ParcelableFileCache<ParcelableKeyRing> cache =
                 new ParcelableFileCache<>(mKeychainService, "key_import.pcl");
```

Comment:
Thank you, I didn't know that. I think I did it just because it was shorter (how it's just four more characters strikes me now). I'll revert it to how it was earlier.